requirements.txt

```diff
@@ -1,4 +1,4 @@
-torch==1.0.1.post2
+torch>=1.1.0
```

Comment:
We are not yet ready to make this migration to 1.1.0. Can you revert this? It causes some issues in data parallel for us, so we need to figure it out properly before making this change.

Reply:
Reverted but that pegging that torch version causes the build to fail.